core/src/main/scala/org/apache/spark/sql/delta/sources/DeltaSQLConf.scala

```diff
@@ -810,6 +810,15 @@ trait DeltaSQLConfBase {
           |""".stripMargin)
       .booleanConf
       .createWithDefault(false)
+
+  val DELTA_CHECK_REQUIRED_SPARK_CONF =
+    buildConf("checkRequiredSparkConf.enabled")
+      .doc(
+      """Whether we check if SparkSession is initialized with spark.sql.extensions
```

Comment:
`Whether to verify SparkSession is initialized with required configurations.` so that we don't need to change this if we add a new required configuration in future.

Reply:
done.